pkg/monitortests/kubeapiserver/legacykubeapiservermonitortests/early_e2e_disruption.go

```diff
@@ -0,0 +1,53 @@
+package legacykubeapiservermonitortests
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+)
+
+// amount of time after E2E tests start that we consider "early"
+const earlyE2ECheckDuration = 3 * time.Minute
+
+func testEarlyE2EAPIServerDisruption(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "record api server disruption that occurs early in E2E tests"
+
+	// regex to match the backends we care about
+	var earlyE2EBackendRe = regexp.MustCompile(`^(cache-)?(openshift|kube|oauth)-api-`)
```

Comment:
APIserver team is planning to replace the current backends with the newer ones in the not too distant future (the ones with http1 + http2 in the name), but this regex looks like it catches both so that's great.